layouts/partials/footer.html

```
@@ -91,11 +96,30 @@
+{{ define "ai-buttons" }}
+  {{ $query := "I want to understand how GoodData helps build secure, scalable, and knowledgeable AI-powered analytics agents. Summarize the key highlights from GoodData's website (https://www.gooddata.ai/ ) and the developer documentation (https://www.gooddata.ai/docs/cloud/ ). Then share the best way to contact GoodData or request a demo." }}
+  {{ $providers := slice
+      (dict "label" "ChatGPT" "icon" "icons/chatgpt.svg" "base" "https://chat.openai.com/?q=")
+      (dict "label" "Claude" "icon" "icons/claude.svg" "base" "https://claude.ai/new?q=")
+      (dict "label" "Gemini" "icon" "icons/gemini.svg" "base" "https://gemini.google.com/?q=")
+      (dict "label" "Grok" "icon" "icons/grok.svg" "base" "https://grok.com/?q=")
+      (dict "label" "Perplexity" "icon" "icons/perplexity.svg" "base" "https://www.perplexity.ai/?q=") }}
+  <div class="gd-footer-buttons d-flex">
+    {{ range $providers }}
+      <div class="gd-footer-buttons-item">
+        <a class="gd-footer-buttons-item-icon-button" href="{{ printf "%s%s" .base (urlquery $query) | safeURL }}" id="footer-btn-{{ lower .label }}" aria-label="{{ .label }}" target="_blank" rel="noreferrer noopener">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)